src/Components/App.js

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import Search from "./Search";
+import Results from "./Results";
+import MoreInfo from "./MoreInfo";
+
+const config = {
+  apiKey: "77c3b516",
+  url: "http://www.omdbapi.com/?i="
+};
+
+class App extends React.Component {
+  constructor(props) {
+    super();
+
+    this.state = {
+      films: [],
+      showDetailedInfo: false,
+      imdbLink: "https://www.imdb.com/title/",
+      imdbID: "",
+      details: {},
+      errorPic: "./assets/woops.jpg"
```

Comment:
if error pic does not change, it could be better off being specified in a config or variable. Reserve state for things that will be changed by your app. Same applies to `imdbLink`